src/shape/bounding-sphere.js

```diff
@@ -28,49 +26,6 @@ class BoundingSphere {
         return lenSq < r * r;
     }
 
-    compute(vertices) {
```

Comment:
did you intend to remove this?

Reply:
Yes. It's literally never worked because it calls a non-existent function of the Vec3 class. Rather than fix the function, since we know nobody is using it, I'd rather delete it to give us a slimmer engine.

Reply:
:+1: